app/controllers/v1/articles_controller.rb

```diff
@@ -5,7 +5,7 @@ def index
     articles = Article.all
     
     if articles.empty? 
-      render_error_message("There are no articles here", 200)
+      render_error_message("There are no articles here", 401)
```

Comment:
Why did you take the approach to send back an error when the database is empty?

Reply:
We have written our code to catch an error, thats why we chose an error status

Reply:
My teammates had some heated discussions about this yesterday... this was what the jury decided